admin_manual/release_notes.rst

```diff
@@ -18,6 +19,113 @@ Release Notes
+
+New options for the occ command to reset user passwords
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The occ command ``user:resetpassword`` allows system administrators to reset or change user passwords. It has been extended to provide the additional options ``--send-email`` and ``--output-link`` which can be used to send a password reset link to the user via mail or to output the password reset link to the command line, respectively. This change is in line with the new local user creation flow which is explained above and can also be used for further processing with scripts. See the ownCloud Documentation and the ``--help`` option for more information.
+
+New default minimum supported Desktop Client version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To ensure a clean and reliable operation of the ownCloud platform it is important to stay up-to-date with the latest releases for the Server as well as the Clients. To take care of compatibility between the Server and Desktop Clients the minimum version the Server will accept connections from has been raised to ``2.3.3``. While it's recommended to keep up with later versions this is the new default value. It can be changed by altering the config.php parameter ``'minimum.supported.desktop.version' => '2.3.3',`` if it really is necessary.
+
+New option to configure the language of mail notifications for public links
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Usually ownCloud renders mail notifications in the language of the recipients, when they are known. For the `recently improved feature <https://doc.owncloud.com/server/10.0/admin_manual/release_notes.html#personal-note-for-public-link-mail-notification>`_ to send public links with a personal note directly from the user interface the recipients' language can't be determined automatically it just knows the recipients' mail addresses. ownCloud therefore uses the language of the user who sent the notification which can have the drawback that recipients can't understand them.
+This is still the default behavior but administrators can now change it via a dropdown menu *Language used for public mail notifications for shared files* in the settings *Sharing* section.
+
+Theming changes
+~~~~~~~~~~~~~~
+
+Mail templates for share notifications do not strip linebreaks from the personal note anymore. This affects the HTML (``core/templates/mail.php``) and plain text (``core/templates/altmail.php``) mail templates. The default templates shipped with ownCloud Server 10.0.10 have been modified to accomodate to these changes. If your custom theme overrides these templates, you have to follow up with the changes:
```

Comment:
s/to accomodate to these changes/to accommodate these changes/

Reply:
addresses, thx